store/sqlstore/team_store.go

```diff
@@ -255,9 +255,6 @@ func (s SqlTeamStore) Get(id string) store.StoreChannel {
 		}
 
 		team := obj.(*model.Team)
-		if len(team.InviteId) == 0 {
```

Comment:
Hmmm... do we need a migration for this bad behavior? ie. If InviteId is empty then generate one?

Reply:
I think that's a good idea, although we automatically generate one in `model/team.go:PreSave` since 28. Oct 2015:
https://github.com/mattermost/mattermost-server/blame/master/model/team.go#L188

I think the likelihood of a team having no InviteID is fairly small, but a migration wouldn't do any harm.

Reply:
Added in last commit